students/Roy_Tate/lesson04/mailroom2.py

```diff
@@ -0,0 +1,184 @@
+# author/student: Roy Tate (githubtater)
+
+from collections import Counter
+
+import sys
+import os
+import os.path
+
+# Create the donor_dict and manually add a few entries
+donor_dict = {}
+donor_dict['Bill'] = [1100, 2100, 3100]
+donor_dict['Mark'] = [1000, 300000.23, 2000]
+donor_dict['Jeff'] = [50, 450.355]
+donor_dict['Roy'] = [4500, 7500, 11221, 30232, 323]
+
+# This is the standard template to be used when sending thank you emails. Ideally this would be saved
+# externally as a template file.
+email_str_ty = '''\n
```

Comment:
Python will retain and preserve new lines in triple quoted strings, so there's no need to use the `\n` character here